Automattic/wp-calypso
# Enable multi-select domains for domain-only flow without a flag and onboarding with a flag

Fixes https://github.com/Automattic/dotcom-forge/issues/4206

## Proposed Changes

* Enables multi-domain selection in the domain step of the `domain-only` flow on all environments without a flag and `onboarding` flow under the `add-multiple-domains-to-cart` flag.

## Testing Instructions

* Confirm multi selection is working properly on `/start/domain/domain-only` without the flag.
* Confirm multi selection is working properly on `/start/domains?flags=domains/add-multiple-domains-to-cart` (with the flag).
* Confirm multi selection is not showing on `/start/domains` (without a flag).
